x-pack/plugins/cloud_security_posture/public/pages/findings/findings_group_by_selector.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiFormLabel, EuiSelect, EuiFormControlLayout, type EuiSelectOption } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { GroupBy } from './findings_container';
+
+interface Props {
+  type: GroupBy;
+  options: EuiSelectOption[];
+  onChange(type: GroupBy): void;
+}
+
+export const FindingsGroupBySelector = ({ type, options, onChange }: Props) => (
+  <EuiFormControlLayout prepend={<GroupByLabel />} style={{ maxWidth: 275 }}>
```

Comment:
is this max-width value supposed to max the widest text option? maybe worth naming this value so we understand what it stands for

Reply:
this used to limit the width so it'll look like the design. switched to the default combobox, so this bit is removed